scripts/requirements-doc.txt

```diff
@@ -3,7 +3,7 @@
 breathe>=4.23.0
 docutils>=0.16
 sphinx>=3.3.0,<3.4.0
-sphinx_rtd_theme
+sphinx_rtd_theme==0.5.2
```

Comment:
Late to the party, but the description says fixed "since" 0.5.2, so I really think this should be `>=0.5.2`.

Since we don't recommend installing in a virtualenv, pinning requirements like this is kind of rude IMO.

Reply:
Fair point, we may actually add `sphinx_rtd_theme>=0.5.2,<1.0`

Reply:
FYI https://github.com/zephyrproject-rtos/zephyr/pull/34108